soc/arm/silabs_sim3/sim3u/soc.h

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2023 GARDENA GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _SOC__H_
+#define _SOC__H_
+
```

Comment:
I know that in some cases on some architectures Zephyr requires an `soc.h` header, even if it's empty. Just wanted to make sure that it is the case here as well.

Reply:
`zephyr/modules/cmsis/cmsis_core_m.h` seems to need it.